docs/platform/howto/create_new_service.rst

```diff
@@ -28,6 +28,8 @@ To create a new service:
 
       This defines the number of servers and what kind of memory, CPU, and disk resources are allocated to your service.
 
+      For **Aiven for Apache Kafka**, **Aiven for PostgreSQL**, and **Aiven for MySQL** services, you can also configure additional storage on top of the amount defined in your selected service plan. This option is available for services hosted on Amazon and Google platforms.
```

Comment:
Should we add a link to the newly added "Add storage space" article here?

Reply:
I wouldn't link to the separate article, because the steps are different (the setting is included in the service creation flow). I've changed the wording a bit to provide a more direct reference to the setting in the UI ;)

Reply:
How will we keep the list of supported clouds updated as this feature rolls out? Will someone be notified somehow? Unless there's a change notification process that I don't know about, I think the long-term accuracy of these docs would be improved if they just had a note that the feature wasn't yet present on all clouds.

Reply:
Sure, that's changed now ;)